pdc/apps/releaseschedule/serializers.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2017 Red Hat
+# Licensed under The MIT License (MIT)
+# http://opensource.org/licenses/MIT
+#
+from rest_framework import serializers
+
+from pdc.apps.release.models import Release
+from pdc.apps.componentbranch.models import SLA
+from pdc.apps.common.fields import ChoiceSlugField
+from pdc.apps.common.serializers import DynamicFieldsSerializerMixin, StrictSerializerMixin
```

Comment:
These mixins should now be imported from `contrib.drf_introspection.serializers`.